CCPP_repro.appBuilder

```diff
@@ -4,7 +4,7 @@
 
 COMPONENTS=( CCPP FMS FV3 )
 
-FV3_MAKEOPT="REPRO=Y CCPP=Y STATIC=Y SUITES=FV3_GFS_2017"
```

Comment:
so we do not need the suite files here?

Reply:
That's correct. If no SUITES=... argument is specified when compile.sh or compile_cmake.sh is called (which is called through the app builder), then all available suites will be compiled into the executable.